README.md

```diff
@@ -74,21 +75,20 @@ Then copy pate directly from there to column 1 in the excel sheet and in excel c
 
 Finally give the name some cool name, the name of the insilicopanel is a very cool name and also makes sense and then sent to the clinician! They will do amazing things with it. 
 
-# NOTE Before doing the below step wait for notification from your clinical contact 
+### NOTE Before doing the below step wait for notification from your clinical contact 
 
 Before doing the below step which would apply the panel for use in clinical routine analysis you need to wait for confirmation from the clinician that the panel has been approved. They will notify you. You don't have to wait until the official "start-date" (införande) but can add it to routine before AS LONG AS IT HAS BEEN APPROVED. 
```

Comment:
Is this part something we should change?

Reply:
Change in what way?